share/functions/__fish_termux_api__complete_group_ids.fish

```diff
@@ -0,0 +1,15 @@
+function __fish_termux_api__complete_group_ids
+    set -l command termux-notification-list
+    set ids ($command | jq --raw-output '.[].group')
+    set titles ($command | jq --raw-output '.[].title')
```

Comment:
`jq` is not guaranteed to be installed, so you would have to check if it is available first.

It's also pretty slow - we've sped up our json reading by using `python` instead.

And this $command is useless, please just call the command directly.